package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "PortfolioCsT1",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "script.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
- [OPTIONAL ] As I mentioned before, the script `start` is missing in your package.json to run your project, see this image, when I try to run `npm start` for this project, it says this 

<img width="536" alt="start" src="https://user-images.githubusercontent.com/79058364/158702390-6953a31e-fed0-445a-abe3-f97cc483342e.png">

- This review is optional but I strongly recommend you take this into consideration and make this change.